### ISSUE TITLE
Review Standard Report

### ISSUE THREAD

**Issue by lucernae** [inasafe/inasafe#3876]
### Problem

<img width="400" alt="screen shot 2017-02-16 at 15 32 59" src="https://cloud.githubusercontent.com/assets/831865/23013480/5eaf5422-f45d-11e6-9461-2d1dead349df.png">

Make this section obey standard.

### Solution

ongoing drafting solutions

**Comment by ivanbusthomi** [comment]
hi @lucernae ,
according to the standard document, this is the changes that I proposed for this ticket. Feel free to add feedback or another suggestion
![image](https://cloud.githubusercontent.com/assets/7351977/23154821/48e82d60-f842-11e6-8243-142d406e1346.png)

**Comment by Gustry** [comment]
@lucernae I still have the `hazard zone` for me (in French), did you remove it already ?

**Comment by lucernae** [comment]
I haven't removed it.
The problem is, we got the label 'Hazard Zone' from hazard classification definition, and I'm not sure on the approach to fix this. We can:

1. Change the 'name' key-value to omit 'zone' in the hazard classification (the easiest). But perhaps it will conflict with other standards, such as in legends or in table attribute, we need to review.

2. Create a substitution rule in report_component metadata for all hazard classification, to customize the header. This way the changes only specific for the report standards. The example is like what I did in https://github.com/inasafe/inasafe/blob/develop/safe/definitions/reports/components.py#L113 to create a 'Not Affected' label, because we don't have definition based text for only 'Not Affected'

Both of them is trivial, I guess. Can @myarjunar take over from here, should be a good issue candidate for exercise 😄 ? We just need to decide which option is the best for the team.
